packages/client/tests/unit/arpa_reporter/helpers/form-helpers.spec.js

```diff
@@ -31,7 +31,7 @@ describe('failing validations', () => {
     const record = { content: '][' };
     const { messages } = validate(columns, record);
     expect(messages[0]).to.equal(
-      'Content Unexpected token ] in JSON at position 0',
+      'Content Unexpected token \']\', "][" is not valid JSON',
```

Comment:
Apparently, the error message returned by `JSON.parse()` changed between Node 18 and Node 20. I assume this change can't be a problem since (1) this is client side code so doesn't run in Node anyway and (2) I can't find any non-test usages of this `validate` function anyway.